docs/octopus-rest-api/octopus-cli/index.md

```diff
@@ -11,35 +11,37 @@ hideInThisSection: true
 
 `octo` supports the following commands:
 
+- **[allow-releaseprogression](/docs/octopus-rest-api/octopus-cli/allow-releaseprogression.md)**: Allows a release to progress to next phase.
 - **[build-information](/docs/octopus-rest-api/octopus-cli/build-information.md)**:  Pushes build information to Octopus Server.
 - **[clean-environment](/docs/octopus-rest-api/octopus-cli/clean-environment.md)**:  Cleans all Offline Machines from an Environment.
 - **[clean-workerpool](/docs/octopus-rest-api/octopus-cli/clean-workerpool.md)**:  Cleans all Offline Workers from a WorkerPool.
-- **[create-autodeployoverride](/docs/octopus-rest-api/octopus-cli/create-autodeployoverride.md)**:  Override the release that auto deploy will use.
+- **[create-autodeployoverride](/docs/octopus-rest-api/octopus-cli/create-autodeployoverride.md)**:  Overrides the release that auto deploy will use.
```

Comment:
This looks like it will be reverted next time the docs generator runs. This is read from the [attribute on the command](https://github.com/OctopusDeploy/OctopusCLI/blob/master/source/Octopus.Cli/Commands/Deployment/CreateAutoDeployOverrideCommand.cs#L15)

Reply:
I see

Reply:
Will create a PR to update the actual command description for consistency